poetry/layouts/layout.py

```diff
@@ -105,9 +120,14 @@ def generate_poetry_content(self):
 
         # Add build system
         build_system = table()
-        build_system_version = ">=" + BUILD_SYSTEM_MIN_VERSION
+        build_system_version = ""
+
+        if BUILD_SYSTEM_MIN_VERSION is not None:
+            build_system_version = ">=" + BUILD_SYSTEM_MIN_VERSION
         if BUILD_SYSTEM_MAX_VERSION is not None:
-            build_system_version += ",<" + BUILD_SYSTEM_MAX_VERSION
+            if build_system_version:
+                build_system_version += ","
+            build_system_version += "<" + BUILD_SYSTEM_MAX_VERSION
```

Comment:
Is there currently a way to set `BUILD_SYSTEM_MIN_VERSION` and `BUILD_SYSTEM_MAX_VERSION`?

Reply:
We do not. The expectation at the moment is that this is dependent on the code (ie. hardcoded if required).

Reply:
In my opinion, than we should just remove it, if we don't need it yet.

Reply:
We will need it in the future if we end up breaking backwards compatibility. We might also be able to extend new/init to support this. Hence why I was hesitant to remove this altogether.